lib/hubspot/meeting.rb

```diff
@@ -7,11 +7,42 @@ class Meeting
     #
     # {https://developers.hubspot.com/docs/api/crm/meetings}
     #
-    CREATE_MEETING_PATH    = '/crm/v3/objects/meetings'
+    MEETINGS_PATH          = '/crm/v3/objects/meetings'
     MEETING_PATH           = '/crm/v3/objects/meetings/:meeting_id'
+    MEETING_SEARCH_PATH    = '/crm/v3/objects/meetings/search'
     ASSOCIATE_MEETING_PATH = '/crm/v3/objects/meetings/:meeting_id/associations/Contact/:contact_id/meeting_event_to_contact'
 
+    BASE_PROPERTIES = %w[hubspot_owner_id hs_meeting_title hs_meeting_body hs_meeting_start_time hs_meeting_end_time].freeze
+
     class << self
+      def all(opts = {})
+        options = { properties: BASE_PROPERTIES.join(','), **opts.compact }
```

Comment:
If you don't pass these base properties, the API doesn't return any meaningful information.

Reply:
But yeah, we could pass properties through opts... instead of hardcoding them in this class, the client could pass what it requires.

Reply:
Given
> If you don't pass these base properties, the API doesn't return any meaningful information.

I think it's a good thing the gem returns something useful. As long as it can be overridden via opts.

Reply:
??

Reply:
If so, it would already work with the above code.

```ruby
(development)> a = { properties: [1234, 4567], after: 123 }
=> {:properties=>[1234, 4567], :after=>123}
(development)> options = { properties: 'will be overriden', **a }
=> {:properties=>[1234, 4567], :after=>123}
```